superbench/benchmarks/micro_benchmarks/cublas_function/cublas_test.cpp

```diff
@@ -0,0 +1,41 @@
+/**
+ * @copyright Copyright (c) Microsoft Corporation
+ * @file cublas_test.cpp
+ * @brief Cublas function benchmark will read the params from cmd, and use these params
+ * to benchmark the wall time of the cublas functions.
+ */
```

Comment:
change to https://docs.opensource.microsoft.com/content/releasing/copyright-headers.html#c-1

Reply:
thank you for the comments